### DISCUSSION TITLE
Displaying images in an image pyramid?

### DISCUSSION THREAD

**Discussion by nancyxyzou** [openseadragon/openseadragon#2504] [Q&A]
Hello! I have images tiles stored in an Azure folder that, together, form a very large image (I believe it is in something like an image pyramid). Is there something in openseadragon that will take the folder containing folders of image tiles and display it in the viewer as one image? Or do I need to do more processing on my end to be able to display the images to the viewer?

Thank you in advance!

**Comment by iangilman** [comment]
Is your image pyramid in a standard format? We support a number of tile source options. 

If not, is it a tiled pyramid (where some of the levels are composed of more than one tile)? If not, you can probably use the legacy tilesource:

https://openseadragon.github.io/examples/tilesource-legacy/

Otherwise, if it's a nonstandard tiled pyramid, you should be able to make it work with a custom tilesource:

https://openseadragon.github.io/examples/tilesource-custom/
